src/windows/Settings/Settings.tsx

```diff
@@ -1094,6 +1094,16 @@ ${t("experimental.tor.disabled.msg2")}`;
     toast(t("msg.written", { ns:namespaces.common }));
   };
 
+  // Bimodal path finding
+  const lndPathfindingAlgorithm = useStoreState((store) => store.settings.lndPathfindingAlgorithm);
+  const changeBimodalPathFindingEnabled = useStoreActions((store) => store.settings.changeLndPathfindingAlgorithm);
+  const changeBimodalPathFindingEnabledPress = async () => {
+    const modal = lndPathfindingAlgorithm === ('apriori' || null) ? 'bimodal' : 'apriori';
```

Comment:
I think this line is incorrect
It should be 

```suggestion
    const modal = (lndPathfindingAlgorithm === 'apriori' || lndPathfindingAlgorithm === null) ? 'bimodal' : 'apriori';
```